test/jest/setup.js

```diff
@@ -3,6 +3,8 @@
 import nock from 'nock';
 import '@testing-library/jest-dom';
 
+process.env.SUPPORT_LINK = 'https://support.metamask.io';
```

Comment:
Can we move this to `test/jest/env.js`... I belive that is where we want to setup all jest tests specific env vars.